src/escpos/printer/win32raw.py

```diff
@@ -60,6 +65,8 @@ class Win32Raw(Escpos):
 
     """
 
+    _device: Union[Literal[False], Literal[None], _win32typing.PyPrinterHANDLE] = False
```

Comment:
```suggestion
    _device: Union[Literal[False], Literal[None], "_win32typing.PyPrinterHANDLE"] = False
```
This should prevent the undefined name

Reply:
I spent yesterday struggling with this issue without success.
Thank you very much for the tip!